package.json

```diff
@@ -164,9 +164,9 @@
     "markdown-it-ruby": "^0.1.1",
     "markdown-it-sub": "~1.0.0",
     "markdown-it-sup": "~1.0.0",
-    "markdownlint": "^0.22.0",
+    "markdownlint": "^0.27.0",
     "markdownlint-rule-helpers": "^0.13.0",
-    "markmap-lib": "^0.4.2",
+    "markmap-lib": "^0.14.4",
```

Comment:
Update to v0.14 seems to fast that needs additional changes, will downgrade to v0.6 so the build won't fail.